Changelog.md

```diff
@@ -12,6 +12,7 @@ Bugfixes:
  * ABIEncoderV2: Fix internal error related to bare delegatecall.
  * ABIEncoderV2: Fix internal error related to ecrecover.
  * ABIEncoderV2: Fix internal error related to mappings as library parameters.
+ * Inline Assembly: Proper error message when ``jump`` refers to a missing jump label.
```

Comment:
Is it just for jumps and not for identifiers in general?

Reply:
Hm, perhaps it is, let me check.

Reply:
Yes, it is for any identifier:
```
function f() {
  assembly {
     x := 1
  }
}
```

Reply:
Updated.